actions/trestle-bot/action.yml

```diff
@@ -81,7 +81,7 @@ outputs:
 
 runs:
   using: "docker"
-  image: "Dockerfile"
+  image: "quay.io/continuouscompliance/trestlebot:latest"
```

Comment:
If we point to the pre-built image, I would prefer we not point to latest. Pointing to latest means that older versions of the trestlebot action could be pointing to the newest image and breaking changes could make older version of the action unusable.